api/geth_backend.go

```diff
@@ -776,6 +793,17 @@ func (b *GethStatusBackend) CreateAccountAndLogin(request *requests.CreateAccoun
 		b.log.Error("start node", err)
 		return err
 	}
+
+	iis, err := images.GenerateIdentityImages(request.ImagePath, 0, 0, 1000, 1000)
```

Comment:
what's iis?

Reply:
images, also I probably need to put those consts somewhere

Reply:
@J-Son89, whether we like it or not, in Go it's idiomatic to use very short names. When looked in isolation, unfortunately these short names can be cryptic and I find necessary to sometimes (eye) jump to the declaration right side to see the actual function name because the variable name is sometimes useless.

There's a very specific method to madness though: https://google.github.io/styleguide/go/decisions#variable-names

Reply:
@cammellos these dimensions `0, 0, 1000, 1000`, seem larger than we normally use, which is fine. FYI we do have dimension consts:

https://github.com/status-im/status-go/blob/45b287370a57b5b9aa374a1c628190c53a41722c/images/meta.go#L13-L26

Also you may want to consider the quality to file size calculations we do with the existing dimensions :

https://github.com/status-im/status-go/blob/45b287370a57b5b9aa374a1c628190c53a41722c/images/meta.go#L34-L43

Reply:
mmh, I will keep the ones used in the frontend for now just to make sure I am not breaking anything, but I will use the const once code has settled, thanks!